macos/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+# Tasks to be run to provision the macOS buildfarm
+
+# Homebrew stuff
+
+# We redirect stdin from /dev/null so it won't prompt for install.
+#- 
+#  name: "Installing Homebrew"
+#  shell: ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null
+# NOTE: Might be best to use https://github.com/geerlingguy/ansible-role-homebrew to avoid prompts etc.
+
+
+- 
+  name: "Installing Homebrew packages"
+  homebrew: 
+    name: "{{ item.name }}"
+  with_items: "{{ brew_packages }}"
```

Comment:
homebrew supports the proxy env vars can add this to this andany other homebrew tasks:

```yaml
  environment:
    http_proxy: "{{ proxy_url | default('') }}"
    https_proxy: "{{ proxy_url | default('') }}"
```